ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContext.scala

```diff
@@ -21,11 +21,12 @@ import scala.collection.mutable
 /** Commit context provides access to state inputs, commit parameters (e.g. record time) and
   * allows committer to set state outputs.
   */
-private[kvutils] trait CommitContext {
+private[kvutils] class CommitContext(
```

Comment:
This could probably be a case class if you wanted, just in case we accidentally log it and it's useful.